### ISSUE TITLE
ShiftIt not working on Ventura (due to interaction with other apps)

### ISSUE THREAD

**Issue by sleeke** [fikovnik/ShiftIt#337]
I know this repo is out of maintenance, so I'm just adding this here to help others, and just maybe find a workaround...

I've noticed that ShiftIt temporarily stops working when using Ventura on my M1 Mac (previously OK on Big Sur).

I've no idea why, but through a process of elimination, I've noticed that the issue happens during the following:

1. Active call on MS Teams (no screensharing)
2. Active call on Slack (no screensharing)

I assume it's something to do with control of the screen only allowed to one app at a time (or similar), but I have no concrete evidence.

The workaround is simple, but annoying: Only use ShiftIt when not doing either of the above.

**Comment by rewolf** [comment]
Yeah for me too it's been working for the most part. Occasionally I had noticed it not working, but didn't correlate it with anything.
But in a Zoom meeting first thing this morning, I was unable to use ShiftIt (my computer had restarted itself over night and windows needed rearranging). But since ending the meeting I have control again.

**Comment by maxencedouet** [comment]
same, only during calls (meet, phone calls transferred on the mac)

**Comment by citadelgrad** [comment]
@sleeke Upgraded recently https://github.com/citadelgrad/ShiftIt
